zetaclient/chains/bitcoin/tx_script.go

```diff
@@ -337,11 +337,11 @@ func decodeInscriptionPayload(t *txscript.ScriptTokenizer) ([]byte, error) {
 // OP_PUSHBYTES_32 <32 bytes> OP_CHECKSIG <Content>
 func checkInscriptionEnvelope(t *txscript.ScriptTokenizer) error {
 	if !t.Next() || t.Opcode() != txscript.OP_DATA_32 {
-		return fmt.Errorf("cannot obtain public key bytes op %d or err %s", t.Opcode(), t.Err())
```

Comment:
I assume the panic was due to `t.Err() == nil` ?

Reply:
It was a panic from call to `t.Opcode()`. When `t.Next()` evaluates to `false`, the opcode inside is a nil.